File: java/src/main/java/com/iteso/nintendo/attacks/Surf.java
package com.iteso.nintendo.attacks;
import com.iteso.nintendo.interfaces.*;

public class Surf implements Attack {
    private int damage = 80;

    @Override
    public String performAttack() {
        return "Surf";
    }

    @Override
    public int getAttackDamage() {
        return this.damage;
    }

    @Override
    public void setAttackDamage(final int damage) {
        this.damage = damage;
    }
}

File: python/src/main/python/Surf.py
from Attack import *

class Surf(Attack):

    def __init__(self, damage = 80):
        self.damage = damage

    def performAttack(self):
        return "Surf"

    def getAttackDamage(self):
        return self.damage

    def setAttackDamage(self, damage):
        self.damage = damage